sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClientOptions.cs

```diff
@@ -65,7 +65,7 @@ public class ServiceBusClientOptions
         /// <value>The default idle timeout is 60 seconds.  The timeout must be a positive value.</value>
         ///
         /// <remarks>
-        ///   If a connection is closed due to being idle, the Event Hubs clients will automatically
+        ///   If a connection is closed due to being idle, the Service Bus clients will automatically
```

Comment:
Should we just say ServiceBusClient?

Reply:
Maybe?  Would that leave someone to doubt that a sender or receiver will have that behavior?   I guess it makes sense since these are SBC options.